sns_ik_lib/utilities/sns_ik_math_utils.cpp

```diff
@@ -284,4 +285,44 @@ bool pseudoInverse(const Eigen::MatrixXd& A, double eps, Eigen::MatrixXd* invA,
 
 /*************************************************************************************************/
 
+bool solveLinearSystem(const Eigen::MatrixXd& A, const Eigen::MatrixXd& b,
+                       Eigen::MatrixXd* x, int* rank, double* res)
```

Comment:
Seems we still have `res` here...

Reply:
My bad!  I did a more thorough find and replace this time.